packages/assets-controllers/src/TokenListController.test.ts

```diff
@@ -657,14 +628,11 @@ describe('TokenListController', () => {
     expect(controller.state.tokenList).toStrictEqual(
       sampleSingleChainState.tokenList,
     );
-    network.setProviderType('goerli');
+    onNetworkStateChangeCallback({ chainId: '5' } as ProviderConfig);
```

Comment:
No need to cast here; we just need one more property to make this a valid argument:
```suggestion
    onNetworkStateChangeCallback({
      chainId: '5',
      type: 'rpc',
    });
```